noxfile.py

```diff
@@ -81,20 +81,17 @@ def install_hydra(session, cmd):
     session.run("pipdeptree", "-p", "hydra-core")
 
 
-def pytest_args(session, check_warning=True, *args):
-    ret = ["pytest"]
-    if check_warning:
-        ret += ["-Werror"]
```

Comment:
noxfile is shared across the core and all plugins. In the final version should be ready to defend any change you made to it.